tonic-build/src/lib.rs

```diff
@@ -4,7 +4,8 @@
 //! # Feature flags
 //!
 //! - `cleanup-markdown`: Enables cleaning up documentation from the generated code. Useful
-//!   when documentation of the generated code fails `cargo test --doc` for example.
+//!   when documentation of the generated code fails `cargo test --doc` for example. `prost`
```

Comment:
I'd suggest adding "The" here: "The `prost` feature [..]".

Reply:
Applied the suggestion.